lib/screens/play/card/card_display.dart

```diff
@@ -42,4 +44,18 @@ class CardDisplay extends StatelessWidget {
         return basicCardColour;
     }
   }
+
+  String _getSubHeaderText(Model.Card card){
+    if (card.questLine != QuestLine.NULL) {
+      return card.questLine.description + " " + _getQuestStage(card);
+    } else if (card.specialType != SpecialType.NULL){
+      return card.specialType.description;
```

Comment:
If we did go for subclasses, this would come out a bit neater I think. Rather than null checking the props we'd just check the type (or, if we want to go for full-on OO, this would be a method on the Card class overridden by each type. So `QuestCard` would have a `getSubHeaderText` implementation on it)